packages/shared/src/components/sidebar/common.tsx

```diff
@@ -142,10 +142,11 @@ export const ItemInner = ({
       <Icon {...item} active={active} />
       <span
         className={classNames(
-          'flex flex-1 flex-row items-center truncate text-left transition-opacity',
+          'flex-1 truncate text-left transition-opacity',
```

Comment:
flex-1 also doesn't do anything anymore right?

Reply:
Don't think so. I'll try to remove it to verify.
It could have an effect as the parent `a` has `flex` on it, but in this case I don't think it actually have any effect.

Reply:
Pushed a change, will verify in preview as soon as it has been deployed.

Reply:
On interesting as it's not even a flex item, but ok let's leave it and go for it :)

Reply:
But the parent is using flex. And `flex-1` is for controlling the flex size of the children of that parent.
![image](https://github.com/dailydotdev/apps/assets/1681525/27e08a01-b1c4-4725-a38c-9d5939bd2dd0)